app/models/hearing.rb

```diff
@@ -55,6 +55,16 @@ def provider_ids
     hearing_body["defenceCounsels"]&.map { |defence_counsel| defence_counsel["id"] }
   end
 
+  def cracked_ineffective_trial
+    return if cp_cracked_ineffective_trial.blank?
+
```

Comment:
could 'cp' be written out in full for extra clarity?

Reply:
Thanks @chris-groves . Normally i'd agree but `cp` abbreviation for common platform is used in a fair few  places (list below) in the codebase and is a relatively well known in the context of this repo. I personally prefer the abbreviation but what does @arthurashman @msahilan07 think?

If we change it we should probably be consistent and remove it from other places it is used to.

https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/app/services/maat_link_creator.rb#L10

https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/app/services/maat_link_creator.rb#L23

https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/app/services/representation_order_creator.rb#L13

https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/app/services/representation_order_creator.rb#L18

https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/app/services/laa_reference_unlinker.rb#L15

https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/app/services/laa_reference_unlinker.rb#L23

https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/app/services/defendant_finder.rb#L10

https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/app/services/defendant_finder.rb#L15

...

Reply:
My preference would probably be to write this out, but maybe we should do all instances as part of a different ticket in the future separate from this PR